public/styles/layout.css

```diff
@@ -465,6 +465,7 @@
 	}
 	.request-item {
 		display: flex;
+        flex-direction: column;
```

Comment:
@polsvoice super small nitpick but could you fix the spacing on this line and `text-align` above?

Reply:
That's odd. The spacing looks fine on my system. Try it now.

Reply:
@polsvoice It still shows misaligned unfortunately. What code editor are you using? Would it be possible for you to integrate Prettier formatting? https://prettier.io/ I have VSCode auto format on save which I think would be a good thing for us to do to keep formatting consistent across files.

Reply:
I'm using Geany. I think I know what the issue is. I have it set up to use 4 spaces instead of tabs. So, while the lines look aligned on my screen, in reality one line has 2 tabs and another has 8 spaces. So, they're going to look differently depending upon what your tab width is. (This is why I prefer to use spaces instead of tabs.)

Reply:
Okay, I changed the spaces to tabs, so it should align now.